products/PurePhone/sys/SystemManager.cpp

```diff
@@ -134,8 +134,6 @@ namespace sys
     {
         SystemManagerCommon::batteryNormalLevelAction();
         CellularServiceAPI::ChangeModulePowerState(this, cellular::service::State::PowerState::On);
-        auto battNormalMsg = std::make_shared<CriticalBatteryLevelNotification>(false);
-        bus.sendUnicast(std::move(battNormalMsg), service::name::appmgr);
```

Comment:
This will break low level battery notifications for Pure -> it will never recover from the low battery screen

Reply:
It also triggers a crash during system start  under specific conditions. In a such scenario there is virtually no possibility to turn on the phone.

Reply:
Instead of deleting it, please report the bug with clear reproduction. TBH I have never observed crash caused by it.